charts/sysdig-deploy/templates/_helpers.tpl

```diff
@@ -35,7 +33,5 @@ Determine sysdig secure endpoint based on provided region
         {{- "eu1.app.sysdig.com/secure" -}}
     {{- else if (eq .Values.global.sysdig.region "au1") -}}
         {{- "app.au1.sysdig.com/secure" -}}
-    {{- else -}}
-        {{- fail (printf "global.sysdig.region=%s provided is not recognized." .Values.global.sysdig.region ) -}}
```

Comment:
Same comment here as above. Could we add an if clause for custom region and just print out that we've been given a custom region?